location.te

```diff
@@ -7,6 +7,9 @@ allow location sysfs_soc:dir rw_dir_perms;
 allow location sysfs_soc:file r_file_perms;
 allow location sysfs_msm_subsys:dir rw_dir_perms;
 allow location sysfs_msm_subsys:file r_file_perms;
+allow location qmuxd_socket:dir rw_dir_perms;
```

Comment:
w_dir_perms should be sufficient according to Luk1337

Reply:
second macro yeah r and w are ok for search but if I am not wrong I saw a read permission too but I forgot to copy/pasta it

Reply:
Ok please add the read deny if you find it